libraries/chain/include/eosio/chain/wasm_interface_private.hpp

```diff
@@ -115,7 +115,6 @@ struct eosvmoc_tier {
       void async_compile_complete(boost::asio::io_context& ctx, const digest_type& code_id, fc::time_point queued_time) {
          if (executing_code_hash.load() == code_id) { // is action still executing?
             auto elapsed = fc::time_point::now() - queued_time;
-            ilog("EOS VM OC tier up for ${id} compile complete ${t}ms", ("id", code_id)("t", elapsed.count()/1000));
```

Comment:
Fine to remove it. Could have made it a `tlog`. It was nice when testing as I could easily see how long it took for the queued up compiles to finish.